qtconsole/mainwindow.py

```diff
@@ -112,6 +112,16 @@ def create_tab_with_current_kernel(self):
             name = '(%s) slave' % current_widget_name
         self.add_tab_with_frontend(widget,name=name)
 
+    def set_tab_title(self):
+        """Set the title of the current tab"""
+        old_title = self.tab_widget.tabText(self.tab_widget.currentIndex())
+        title, ok = QtGui.QInputDialog.getText(self,
+                                               "Rename Tab",
+                                               "New Title:".format(old_title),
```

Comment:
I still think the label should read "New title:" and not "New Title", there is no reason to use title casing here.